rmc/resources/reference_data.py

```diff
@@ -13,37 +17,62 @@
 ####################################################################################
 ## Reference genome related resources
 ####################################################################################
-GENCODE_VERSION = "19"
+GENCODE_VERSION = "39"
 """
 GENCODE version used to annotate variants.
 
 gnomAD v2 used GENCODE v19 and VEP v85.
+gnomAD v4 used GENCODE v39 and VEP v105.
 See: https://gnomad.broadinstitute.org/help/what-version-of-gencode-was-used-to-annotate-variants.
 """
 
-VEP_VERSION = "85"
+VEP_VERSION = "105"
 """
 VEP version used to annotate variants.
 """
 
-gene_model = TableResource(path=f"{RESOURCE_BUILD_PREFIX}/browser/b37_transcripts.ht")
+gene_model = VersionedTableResource(
+    default_version=CURRENT_BUILD,
+    versions={
+        "GRCh37": TableResource(
```

Comment:
`RESOURCE_BUILD_PREFIX` and `REF_DATA_PREFIX` both refer to `CURRENT_BUILD` which is GRCh38 even for versioned table resources that correspond to GRCh37 - fix?